### PR TITLE
Merge packages

### PR DESCRIPTION
**Pull request by woksin**
## Summary

Merge the grpc contracts packages into one

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -367,3 +367,4 @@ Generation/Go/*
 !Generation/Go/generate.sh
 
 Generation/Artifacts
+!Generation/CSharp/VersionInfo.cs
```
Comment (jakhog) [inline .gitignore]: Why is this needed? If we have to do this - wouldn't the same need to apply for all languages?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -176,39 +170,24 @@ jobs:
...
           regex: '^\d+\.\d+.\d+-([^.]+)\.\d+$'
-      - name: Publish Fundamentals
-        working-directory: ./Generation/JavaScript/Fundamentals
-        run: npm publish --tag ${{ needs.context.outputs.release-type == 'prerelease' && steps.tag_regex.outputs.group1 || 'latest' }}
-        env:
-          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
-      - name: Publish Runtime
-        working-directory: ./Generation/JavaScript/Runtime
+      - name: Publish Contracts
+        working-directory: ./Generation/JavaScript/Contracts
```
Comment (jakhog) [inline .github/workflows/ci.yml]: Might get the answer further down - but any particular reason there is a Contracts package inside here?

Comment (jakhog) [inline .github/workflows/ci.yml]: Might get the answer further down - but any particular reason there is a Contracts package inside here?

File: Generation/CSharp/Contracts.csproj [hunk 3]
```diff
@@ -1,12 +1,12 @@
 <Project Sdk="Microsoft.NET.Sdk">
-    <Import Project="../../../default.props"/>
+    <Import Project="../../default.props"/>
```
Comment (jakhog) [inline Generation/CSharp/Contracts.csproj]: If we just have one `.csproj` now - maybe it is not particularly useful to have the `default.props` file anymore?

Comment (jakhog) [inline Generation/CSharp/Contracts.csproj]: If we just have one `.csproj` now - maybe it is not particularly useful to have the `default.props` file anymore?

File: Generation/JavaScript/Contracts/package.json [hunk 4]
```diff
@@ -20,7 +20,7 @@
         "access": "public"
     },
     "scripts": {
-        "build": "dolittle_proto_build grpc-node --skip-empty-files -O. -RFundamentals/: -I../../../Source ../../../Source/Fundamentals",
+        "build": "dolittle_proto_build grpc-node --skip-empty-files -O. -RContracts/: -I../../../Source ../../../Source",
```
Comment (jakhog) [inline Generation/JavaScript/Contracts/package.json]: Lets try to get rid of this script.

Comment (jakhog) [inline Generation/JavaScript/Contracts/package.json]: Lets try to get rid of this script.

File: Generation/JavaScript/package.json [hunk 5]
```diff
@@ -2,8 +2,7 @@
   "private": true,
   "workspaces": [
     "Build",
-    "Fundamentals",
-    "Runtime"
+    "Contracts"
```
Comment (jakhog) [inline Generation/JavaScript/package.json]: Could we just get rid of this Workspace setup now that there is really just one package?

Comment (jakhog) [inline Generation/JavaScript/package.json]: Could we just get rid of this Workspace setup now that there is really just one package?

File: Generation/JavaScript/package.json [hunk 6]
```diff
@@ -2,8 +2,7 @@
...
   ],
   "devDependencies": {
     "typescript": "^3.8.3"
```
Comment (jakhog) [inline Generation/JavaScript/package.json]: Let's also go through dependencies and make them pinned to versions

Comment (jakhog) [inline Generation/JavaScript/package.json]: Let's also go through dependencies and make them pinned to versions

File: Source/Embeddings/Store.proto [hunk 7]
```diff
@@ -3,15 +3,15 @@
...
+import "Services/CallContext.proto";
+import "Projections/State.proto";
 
-package dolittle.runtime.embeddings;
+package dolittle.embeddings;
 
-option csharp_namespace = "Dolittle.Runtime.Embeddings.Contracts";
-option go_package = "go.dolittle.io/contracts/runtime/embeddings";
+option csharp_namespace = "Dolittle.Embeddings.Contracts";
+option go_package = "go.dolittle.io/contracts/embeddings";
```
Comment (jakhog) [inline Source/Embeddings/Store.proto]: Not at all related to this file - but the Go scripts might be very tied to the old structure and broken now.

Comment (jakhog) [inline Source/Embeddings/Store.proto]: Not at all related to this file - but the Go scripts might be very tied to the old structure and broken now.

File: Source/Events.Processing/StreamEvent.proto [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+
+import "Protobuf/Uuid.proto";
+import "Events/Committed.proto";
+
+package dolittle.events.processing;
+
+option csharp_namespace = "Dolittle.Events.Processing.Contracts";
+option go_package = "go.dolittle.io/contracts/events/processing";
+
+message StreamEvent {
```
Comment (jakhog) [inline Source/Events.Processing/StreamEvent.proto]: What happened here? Was this in some other file before? Surprised about this being a brand new file...

Comment (jakhog) [inline Source/Events.Processing/StreamEvent.proto]: What happened here? Was this in some other file before? Surprised about this being a brand new file...

**Review by jakhog — CHANGES_REQUESTED** [review]
Some things mostly to remember to fix today. Let's figure out what we want to do with the package and namespaces together.

File: .gitignore (hunk 1 above)
Comment (woksin) [reply .gitignore]: Yes, I did not understand this either

**Review by jakhog — APPROVED** [review]
